angular/demo/package.json

```diff
@@ -24,7 +24,7 @@
         "@pxblue/angular-components": "2.0.0-beta.9",
         "@pxblue/colors": "3.0.0",
         "@pxblue/ng-progress-icons": "1.1.4",
-        "@pxblue/angular-themes": "latest",
+        "@pxblue/angular-themes": "^5.0.0-beta.8",
```

Comment:
the demo should always run off latest. It might cause a temporary issue until we have the new package published, but that's intentional.